examples/ts/proxy/create-wallet.ts

```diff
@@ -0,0 +1,33 @@
+import { BitGo } from 'bitgo';
+
+// This script emulates a front-end using the BitGo SDK.
+// Set up the BitGo connection object.
+const bitgo = new BitGo({
+  // This value is ignored in favor of your real developer token, stored in your backend proxy machine.
+  accessToken: 'unusedValue',
+  // Set as prod/test as needed for whatever BitGo environment you want to use.
+  // This *must* match the BitGo platform API your proxy instance is using.
+  env: 'test',
+  // In your real setup this would be <your.custom.backend.urlroot>, where you host the proxy instance.
+  customRootURI: 'localhost:3000',
+});
+
+async function createHotMultiSigWalletSimple() {
```

Comment:
dont we already have this in our examples

Reply:
how is this different from https://github.com/BitGo/BitGoJS/blob/3e6b9a0309a2d12fed17877a04efaf418adba2c9/examples/ts/create-wallet.ts

Reply:
This is just a packaged example to let a client run it more or less out of the box.

Reply:
You could add an export to the other example and import it here if you want to avoid duplication.